src/lib/mina_base/parties_logic.ml

```diff
@@ -453,6 +457,14 @@ module type Account_intf = sig
   val token_symbol : t -> token_symbol
 
   val set_token_symbol : token_symbol -> t -> t
+
+  val public_key : t -> public_key
+
+  val set_public_key : public_key -> t -> t
+
+  val delegate : t -> public_key
+
+  val set_delegate : public_key -> t -> t
```

Comment:
do we need this?

Reply:
I'm not sure I understand; it's explicitly used [below](https://github.com/MinaProtocol/mina/pull/10223/files#diff-590580f666541f8fd7a9952377d5a523e8cb153e9d42c62ed55842cac8246bb0R1095).